EIPS/eip-666.md

```diff
@@ -0,0 +1,52 @@
+---
+eip: 666
+title: The Number of the Beast
+description: Two horns.
+author: Beelzebub (@בַּע-זְבוּב)
+discussions-to: https://hell.com
+status: Undead
```

Comment:
Doesn't this imply that this was once a living standard?

Reply:
At the limit in infinity, yes.